moped-editor/src/utils/activityLogFormatters/mopedFundingActivity.js

```diff
@@ -10,46 +10,61 @@ export const formatFundingActivity = (
   const entryMap = ProjectActivityLogTableMaps["moped_proj_funding"];
 
   const changeIcon = <MonetizationOnOutlinedIcon />;
-  let changeDescription = "Project funding updated";
-  let changeValue = "";
+  const changeText = [];
 
   // add a new funding source
   if (change.description.length === 0) {
-    changeDescription = "Added a new funding source";
     // if the added record has a funding source, use that as the change value
     if (change.record_data.event.data.new.funding_source_id) {
-      changeValue =
-        fundingSources[change.record_data.event.data.new.funding_source_id];
+      changeText.push({ text: "Added ", style: null });
+      changeText.push({
+        text: fundingSources[
+          change.record_data.event.data.new.funding_source_id
+        ],
+        style: "boldText",
+      });
+      changeText.push({ text: " as a new funding source.", style: null });
```

Comment:
instead of doing all this pushing, you could `return` from the if/else blocks throughout?

```javascript
  if (change.record_data.event.data.new.funding_source_id) {
      return {
        changeIcon,
        changeText: [
          { text: "Added ", style: null },
          {
            text: fundingSources[
              change.record_data.event.data.new.funding_source_id
            ],
            style: "boldText",
          },
          { text: " as a new funding source.", style: null },
        ],
      };
```